simple_parsing/wrappers/dataclass_wrapper.py

```diff
@@ -12,19 +12,20 @@
 
 logger = getLogger(__name__)
 
+
 class DataclassWrapper(Wrapper[Dataclass]):
     def __init__(self,
                  dataclass: Type[Dataclass],
                  name: str,
-                 default: Dataclass=None,
-                 prefix: str="",
-                 parent: "DataclassWrapper"=None,
-                 _field: dataclasses.Field=None,
+                 default: Union[Dataclass, Dict] = None,
```

Comment:
Neat! Would you mind also changing the type annotation for the `default` argument of `ArgumentParser.add_arguments` ([found here](https://github.com/idoby/SimpleParsing/blob/34a33172c7235f72be1ec30f31e0741e376c0759/simple_parsing/parsing.py#L108)) as well? Not a big deal though.

Reply:
Done. PyCharm did some extra formatting, which I left in this time. Feel free to revert the cosmetic changes